gwt-numberformat/src/main/resources/org/gwtproject/i18n/NumberFormat.gwt.xml

```diff
@@ -0,0 +1,6 @@
+<module>
+    <inherits name="elemental2.core.Core"/>
+    <inherits name="elemental2.dom.Dom"/>
+    <source path="client"/>
+    <source path="shared"/>
+</module>
```

Comment:
if we're using the ltgt plugin, this should be module.gwt.xml, right? and only need the two elemental2 inherits

Reply:
even though we are using the plugin i didnt add the gwt.xml under the main folder and only doing it manually.